src/docfx/restore/FileResolver.cs

```diff
@@ -178,7 +180,13 @@ private Task<HttpResponseMessage> GetAsync(string url, EntityTagHeaderValue etag
 
                 _provideCredential?.Invoke(message);
 
-                return s_httpClient.SendAsync(message);
+                var response = await OpsConfigAdapter.InterceptHttpRequest(_errorLog, message);
```

Comment:
>OpsConfigAdapter.InterceptHttpRequest( [](start = 37, length = 38)

minor: It feels that this bring extra buisiness into a lower-level functionality.